### ISSUE TITLE
Incorrect cache folder path in show_diagnostics

### ISSUE THREAD

**Issue by artemiyokulov** [tj/n#795]
# Bug Report

https://github.com/tj/n/blob/v9.2.0/bin/n#L1561

Instead `local cache_root="${N_PREFIX}/n"` needs `local cache_root="${N_CACHE_PREFIX}/n"`

## Steps to Reproduce

1. Set `N_CACHE_PREFIX=/blabla/any/path` 
2. set  `N_PREFIX=/some/other/folder`
3. run `mkdir -p $N_CACHE_PREFIX $N_PREFIX && n doctor`

## Expected Behaviour

doctor command run successful

## Actual Behaviour

get error 

Checking permissions for cache folder...
INFO Cache folder does not exist: /usr/local/n

## Configuration Details

```bash
$ n --version

9.2.0

$ command -v node
?

$ node -p process.platform
?
```

**Comment by shadowspawn** [comment]
Oops. The directory checks in "doctor" did not get updated when the possible separate cache folder got added in #717

**Comment by shadowspawn** [comment]
Thanks for report. Fixed in v9.2.1.
